modules/delegation-renewal/hooks/useDelegatedToExpired.tsx

```diff
@@ -0,0 +1,23 @@
+import { add, isBefore } from 'date-fns';
+import useDelegatedTo from 'modules/delegates/hooks/useDelegatedTo';
+import { useActiveWeb3React } from 'modules/web3/hooks/useActiveWeb3React';
+
+export default function useDelegatedToExpired(): boolean {
+  const { account: address, network } = useActiveWeb3React();
+
+  const { data: delegatedToData } = useDelegatedTo(address, network);
```

Comment:
Would it make more sense to add this check on about to expire to the `delegated-to` endpoint and then it would be part of the `useDelegatedTo` hook too and then just use that everywhere? It's just adding one more properly from reducing an array. But then it'd get cached with the API endpoint and SWR as well.

Reply:
idk, seems like a really frontend logic to me, the api is giving us a expiryDate, that should be enough, having also "isAboutToExpire" or "isExpired" seems redundant information